.circleci/docker/common/install_conda.sh

```diff
@@ -75,7 +75,7 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   }
 
   # Install PyTorch conda deps, as per https://github.com/pytorch/pytorch README
-  CONDA_COMMON_DEPS="astunparse pyyaml mkl=2022.0.1 mkl-include=2022.0.1 setuptools cffi six"
+  CONDA_COMMON_DEPS="astunparse pyyaml mkl=2022.0.1 mkl-include=2022.0.1 setuptools six"
```

Comment:
Actually, is six still needed? I think it was there for python2

Reply:
I don't think it is strictly needed no as we don't need these constructs anymore. But it is used quite a bit in the codebase. So we would need to remove use of it first before being able to remove it from the builds.

Reply:
will create an issue for dropping it eventually